boards/arm/disco_l475_iot1/support/openocd.cfg

```diff
@@ -0,0 +1,7 @@
+source [find interface/stlink-v2-1.cfg]
+
+transport select hla_swd
+
+source [find target/stm32l4x.cfg]
+
+reset_config srst_only
```

Comment:
For most of the openocd.cfg we have, curious if we dont need it:

$_TARGETNAME configure -event gdb-attach {
        echo "Debugger attaching: halting execution"
        reset halt
        gdb_breakpoint_override hard
}

$_TARGETNAME configure -event gdb-detach {
        echo "Debugger detaching: resuming execution"
        resume
}

Reply:
Actually, I don't even know how this is called/used, "make debug" does not seem to require those.

Reply:
I'm not sure -- I obtained this file from Christophe directly. Perhaps Erwan can comment.